### DISCUSSION TITLE
Querying Regex with IgnoreCase option not working...

### DISCUSSION THREAD

**Discussion by ashokgudur** [ravendb/ravendb#13636] [General]
Hi,

The following is not working
_session.Query<GstinMaster>()
             .Where(x => Regex.IsMatch(x.Gstin, regex, RegexOptions.IgnoreCase))
             .ToList();

However the following working:
_session.Query<GstinMaster>()
             .Where(x => Regex.IsMatch(x.Gstin, regex))
             .ToList();

Error:
NotSupportedException: Regex.IsMatch() overload with 3 arguments is not supported. Only Regex.IsMatch(string input, string pattern) overload is supported.Expression: IsMatch(x.Gstin, value(Aqume.Carbon.Dao.GstinMasterDao+<>c__DisplayClass3_2).regex, IgnoreCase).

**Comment by garayx** [comment]
Hi 

Currently `Regex.IsMatch()` overload with _only_ 2 arguments is supported.

I have opened an issue to support the 3 arguments implementation, 

https://issues.hibernatingrhinos.com/issue/RavenDB-18029
